collections.html

```diff
@@ -155,21 +155,23 @@
 				<div property="numbers" mv-multiple style="--prepend: '[numbers]'; --append: '[$index]';"></div>
 			</td>
 			<td>
-				<div mv-multiple="" style="--prepend: '1'; --append: '0'">1</div><!--
-				--><div mv-multiple="" style="--prepend: '2'; --append: '1'">2</div><!--
-				--><div mv-multiple="" style="--prepend: '3'; --append: '2'">3</div><!--
-				--><div mv-multiple="" style="--prepend: '4'; --append: '3'">4</div><!--
-				--><div mv-multiple="" style="--prepend: '5'; --append: '4'">5</div><!--
-				--><div mv-multiple="" style="--prepend: '6'; --append: '5'">6</div><!--
-				--><div mv-multiple="" style="--prepend: '7'; --append: '6'">7</div><!--
-				--><div mv-multiple="" style="--prepend: '8'; --append: '7'">8</div><!--
-				--><div mv-multiple="" style="--prepend: '9'; --append: '8'">9</div><!--
-				--><div mv-multiple="" style="--prepend: '10'; --append: '9'">10</div>
+				<div mv-multiple="" style="--prepend: '-1'; --append: '0'">-1</div><!--
+				--><div mv-multiple="" style="--prepend: '0'; --append: '1'">0</div><!--
+				--><div mv-multiple="" style="--prepend: '1'; --append: '2'">1</div><!--
+				--><div mv-multiple="" style="--prepend: '2'; --append: '3'">2</div><!--
+				--><div mv-multiple="" style="--prepend: '3'; --append: '4'">3</div><!--
+				--><div mv-multiple="" style="--prepend: '4'; --append: '5'">4</div><!--
+				--><div mv-multiple="" style="--prepend: '5'; --append: '6'">5</div><!--
+				--><div mv-multiple="" style="--prepend: '6'; --append: '7'">6</div><!--
+				--><div mv-multiple="" style="--prepend: '7'; --append: '8'">7</div><!--
+				--><div mv-multiple="" style="--prepend: '8'; --append: '9'">8</div><!--
+				--><div mv-multiple="" style="--prepend: '9'; --append: '10'">9</div><!--
+				--><div mv-multiple="" style="--prepend: '10'; --append: '11'">10</div>
 			</td>
```

Comment:
Any idea why this diff is so big? It looks like it should only be two added lines.

Reply:
The --append to my knowledge represents the index, and since I added 2 elements to the beginning, the rest of the indices were shifted by 2.